### PR TITLE
 Removing *subject_ids* argument in TUHAbnormal example.

### PR DESCRIPTION
**Pull request by bruAristimunha**
Related with the issue #377.

Removing a deprecated parameter in the TUHAbnormal example.

### CODE REVIEW

**Comment by robintibor** [comment]
So is this correct @gemeinl ?

**Comment by gemeinl** [comment]
No, actually not correct. 
The function `load_example_data` has an argument `subject_ids` which does not have any effect anymore, due to the changes introduced here. Therefore, the entire dataset will be loaded instead of just 10 recs.
We still need this argument, however, its meaning has changed. It is not subjects we are refering to but recordings. I suggest to rename the argument to `n_recordings` and give this as input to `TUHAbnormal` which has the same argument (see https://github.com/braindecode/braindecode/blob/884a78b2fb306047d444352ac4dcd9b503324fca/braindecode/datasets/tuh.py#L207-L208).

**Comment by bruAristimunha** [comment]
Many thanks for the explication @gemeinl. I was wondering, can you review it again to make sure everything is correct?

File: examples/benchmark_lazy_eager_loading.py [hunk 1]
```diff
@@ -68,15 +68,17 @@
 # Each one of these steps will be timed, so we can report the total time taken
 # to prepare the data and train the model.
 
-def load_example_data(preload, window_len_s, n_subjects=10):
+def load_example_data(preload, window_len_s, n_recordings=[10]):
```
Comment (gemeinl) [inline examples/benchmark_lazy_eager_loading.py:71]: `n_recordings=10`

File: examples/benchmark_lazy_eager_loading.py [hunk 2]
```diff
@@ -89,7 +91,8 @@ def load_example_data(preload, window_len_s, n_subjects=10):
         resampled to a common sampling rate.
     """
```
Comment (gemeinl) [inline examples/benchmark_lazy_eager_loading.py:92]: `recording_ids=list(range(n_recordings))`

File: examples/benchmark_lazy_eager_loading.py [hunk 3]
```diff
@@ -89,7 +91,8 @@ def load_example_data(preload, window_len_s, n_subjects=10):
...
     ds = TUHAbnormal(
-        TUH_PATH, target_name='pathological',
+        TUH_PATH, recording_ids=n_recordings,
```
Comment (gemeinl) [inline examples/benchmark_lazy_eager_loading.py:94]: `recording_ids=recording_ids`

**Comment by gemeinl** [comment]
I suggest to stick as close to the original code as possible.
So change the input argument `n_subject` to `n_recordings` in `load_example_data` and create the `recorind_ids` for `TUHAbnormal` inside as before.

**Comment by bruAristimunha** [comment]
Done @gemeinl.

**Review by gemeinl — APPROVED** [review]
Great, thanks a lot @bruAristimunha !

File: examples/benchmark_lazy_eager_loading.py [hunk 4]
```diff
@@ -216,7 +221,7 @@ def run_training(model, dataloader, loss, optimizer, n_epochs=1, cuda=False):
 # Next, we define the different hyperparameters that we want to compare:
 
 PRELOAD = [True, False]  # True -> eager loading; False -> lazy loading
-N_SUBJECTS = [10]  # Number of recordings to load from the TUH Abnormal corpus
+N_RECORDINGS = [10]  # Recording ids to load from the TUH Abnormal corpus
```
Comment (robintibor) [inline examples/benchmark_lazy_eager_loading.py:224]: Is this now recording ids or number of recordings?

Comment (bruAristimunha) [reply examples/benchmark_lazy_eager_loading.py]: Thank you, @robintibor. You are correct. I updated the comment.

Comment (bruAristimunha) [reply examples/benchmark_lazy_eager_loading.py]: Number of recordings

**Comment by bruAristimunha** [comment]
Can you merge? @robintibor

**Comment by cedricrommel** [comment]
Could you solve the conflicts @bruAristimunha before I merge?

**Comment by bruAristimunha** [comment]
Done @cedricrommel.

**Pull request merged**